services/management/checks.go

```diff
@@ -49,3 +49,23 @@ func (s *ChecksAPIService) StartSecurityChecks(ctx context.Context, request *man
 
 	return &managementpb.StartSecurityChecksResponse{}, nil
 }
+
+// GetSecurityCheckResults returns the results of the STT checks that were run.
+func (s *ChecksAPIService) GetSecurityCheckResults(ctx context.Context, request *managementpb.GetSecurityCheckResultsRequest) (*managementpb.GetSecurityCheckResultsResponse, error) {
+	results := s.checksService.GetSecurityCheckResults(ctx)
+	if len(results) == 0 {
+		return nil, status.Error(codes.Internal, "Failed to get check results.")
```

Comment:
Why empty results is error?

Reply:
because the generated code has that method signature `(response, error)` https://github.com/percona/pmm/pull/560/files#diff-415f5bc55c8bafbe115393120e823e80R348

so it won't compile if I change the signature from pmm-managed side by removing error, any idea how to make protobuf generate this method without returning error? @artemgavrilov

Reply:
I mean why not?
```suggestion
		return &managementpb.GetSecurityCheckResultsResponse{}, nil
```
Empty results is not an error, it's just an empty slice.

Reply:
This is doable :point_up: 

I misunderstood earlier, I thought you wanted me to change the function signature and just return the response

Reply:
So this block can be removed

Reply:
yup, will do this in the next push and also add an error in case STT is disabled which I forgot to add initially